.taskcat.yml

```diff
@@ -1,5 +1,5 @@
 project:
-  name: cfn-ps-sumo-logic-cloudsiem
+  name: cfn-ps-sumo-logic-cloudsiem/2
```

Comment:
please remove "/2" from project name in taskcat

Reply:
Also uncomment and change lines 19-22 so we can use in the automated tests by referencing SSM Parameter Store for these variables in our test AWS Account:

Section1SumoLogicDeployment: "us2"
Section1SumoLogicAccessID: $[taskcat_ssm_/sumologic/Section1SumoLogicAccessID]
Section1SumoLogicAccessKey: $[taskcat_ssm_/sumologic/Section1SumoLogicAccessKey]
Section1SumoLogicOrganizationId: $[taskcat_ssm_/sumologic/Section1SumoLogicOrganizationId]